Assets/Scripts/Models/Furniture.cs

```diff
@@ -132,9 +132,9 @@ private Furniture(Furniture other)
         UnlocalizedDescription = other.UnlocalizedDescription;
     }
 
-    public event Action<Furniture> Changed;
+    public event Action<Furniture> OnChanged;
 
-    public event Action<Furniture> Removed;
+    public event Action<Furniture> OnRemoved;
```

Comment:
It might be a jQuery thing where the register methods start with on, and now it feels like a good name mainly because there isn't a Register method and it is used like ```Furniture.OnRemoved += function```.